src/ioc/Ioc.js

```diff
@@ -0,0 +1,17 @@
+export class Ioc {
+    constructor() {
+        this.services = {};
+    }
+
+    register(service, serviceName) {
+        if (this.services[serviceName]) {
+            console.log("Service already created.")
+        } else {
+            this.services[serviceName] = service;
+        }
+    }
+
+    getService(serviceName) {
```

Comment:
specify the method is public. Also please throw and error if `serviceName` is falsy

Reply:
Some here I think you can also simplify this here